### PR TITLE
Add instructions for a standard kubernetes setup

### PR DESCRIPTION
**Pull request by worldtiki**
### :pencil: Description
Instructions for a standard kubernetes setup

### CODE REVIEW

File: docs/installation/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        release: pitchfork
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8081"
+        prometheus.io/path: "/actuator/prometheus"
+    spec:
+      terminationGracePeriodSeconds: 0
+      containers:
+        - name: pitchfork
+          image: expediagroup/pitchfork:1.19
```
Comment (nikos912000) [inline docs/installation/kubernetes.md:41]:
```suggestion
          image: expediagroup/pitchfork:latest
```

File: docs/installation/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          ports:
+            - containerPort: 9411
+              name: pitchfork
+            - containerPort: 8081
+              name: actuator
+          env:
+            - name: SERVER_PORT
+              value: "9411"
+            - name: JAVA_JVM_ARGS
+              value: "-Dspring.jmx.enabled=true -XX:MaxRAMPercentage=80.0"
```
Comment (nikos912000) [inline docs/installation/kubernetes.md:51]: I see we've [enabled JMX](https://github.com/ExpediaGroup/pitchfork/blob/cd41d627f6c5fbedaedc520c8bb740ba3adbd90d/src/main/resources/application.yml#L72-L73) for metrics, but do we really need it?
Setting `spring.jmx.enabled` to true adds some overhead (see [here](https://github.com/spring-projects/spring-boot/issues/16090#issuecomment-469635778)) and wondering if it's useful or not.

File: docs/installation/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            - name: PITCHFORK_FORWARDERS_LOGGING_ENABLED
+              value: "true"
+            # You can enabled and configure more forwarders here.
```
Comment (nikos912000) [inline docs/installation/kubernetes.md:54]:
```suggestion
            # You can enable and configure more forwarders here.
```

File: docs/installation/kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    metadata:
+      labels:
+        app: pitchfork
+        release: pitchfork
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8081"
+        prometheus.io/path: "/actuator/prometheus"
+    spec:
+      terminationGracePeriodSeconds: 0
```
Comment (nikos912000) [inline docs/installation/kubernetes.md:38]: I think this needs to be > preStop hook sleep duration. By default it's 30s.
Kubernetes waits for time=terminationGracePeriodSeconds and this happens in parallel to whatever we set in the preStop hook.

File: website/sidebars.json [hunk 5]
```diff
@@ -1,15 +1,15 @@
...
-      "installation/building",
-      "installation/running"
-
+    "Running": [
+      "installation/running",
+      "installation/configuration-options",
+      "installation/kubernetes"
     ],
     "Contributing": [
+      "contributing/building",
```
Comment (nikos912000) [inline website/sidebars.json:12]: This md doesn't exist.
Don't you want to keep the [section](https://github.com/ExpediaGroup/pitchfork/blob/master/docs/installation/building.md) on how to build?

File: docs/installation/kubernetes.md (hunk 4 above)
Comment (worldtiki) [reply docs/installation/kubernetes.md:38]: yes, you're right. i'll remove this line.

File: docs/installation/kubernetes.md (hunk 1 above)
Comment (worldtiki) [reply docs/installation/kubernetes.md:41]: i'll add a comment at the top asking users to replace "latest" with the latest available version at the time.

File: docs/installation/kubernetes.md (hunk 2 above)
Comment (worldtiki) [reply docs/installation/kubernetes.md:51]: the kafka metrics require this to be enabled :(

File: website/sidebars.json (hunk 5 above)
Comment (worldtiki) [reply website/sidebars.json:12]: i'll group them together

File: docs/installation/kubernetes.md (hunk 2 above)
Comment (worldtiki) [reply docs/installation/kubernetes.md:51]: regarding the metrics, there's folks using jmx agents to get metrics.

**Pull request merged**